src/pa02/chutes_simulation.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Aleksander Eriksen', 'Elin Wølner Bjørnson'
+__email__ = 'jaer@nmbu.no', 'elinbj@nmbu.no'
+
+from collections import defaultdict
+from random import randint
+import random
+
+
+class Board:
+    std_goal = 90
+    std_board = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82, 68: 85,
+                 24: 5, 33: 3, 42: 30, 56: 37, 64: 27, 74: 12, 87: 70}
+
+    def __init__(self, ladders=None, chutes=None, goal=None):
+        """
+        Initialise Board
+        Parameters
+        ----------
+        ladders: dict
+            positions and destinations of ladders
+        chutes: dict
+            positions and destinations of chutes
+        goal: int
+            position where the goal is reached
+        """
+
+        if ladders and chutes and goal is not None:
+            self.board = {**dict(ladders), **dict(chutes)}
+            self.goal = goal
+
+        else:
+            self.board = Board.std_board
+            self.goal = Board.std_goal
+
+    def goal_reached(self, position):
+        """
+        Returns true or false if goal is reached
+        Parameters
+        ----------
+        position: int
+
+        Returns
+        -------
+        bool
+        """
+        return position >= self.goal
+
+    def position_adjustment(self, position):
+        """
+        Returns 'delta(Δ) position' if players lands on ladder or chute
+        else it will return 0
+        Parameters
+        ----------
+        position: int
+
+        Returns
+        -------
+        int
+            delta position
+        """
+
+        return self.board.get(position, position) - position
+
+
+class Player:
+    def __init__(self, board=Board()):
+        """
+        Initialise Player
+        Parameters
+        ----------
+        board: class
+            the board used in the game.
+        """
+        self.board = board
+        self.position = 0
```

Comment:
Need number of steps as attribute, which shouldn't be calculate outside this class.